### PR TITLE
Bump @formatjs/intl-datetimeformat

### PR DESCRIPTION
**Pull request by plesiecki**
Hello,
I would like to update the `@formatjs/intl-datetimeformat` dependency as [it polyfills the `Date` class better from version 5.0.2.](https://github.com/formatjs/formatjs/pull/3559)

### CODE REVIEW

**Comment by plesiecki** [comment]
@JakeChampion Thanks for info 👍

**Comment by plesiecki** [comment]
Hello @chee. I'm sorry to bother you again, but may I ask you to run tests on this one too, please?

**Comment by plesiecki** [comment]
Well, it doesn't look like any tests have started.

**Comment by chee** [comment]
> Well, it doesn't look like any tests have started.

it sure doesn't! this is on my todo list for later. i will do it the way i did the other one.

**Comment by chee** [comment]
> > Well, it doesn't look like any tests have started.
> 
> it sure doesn't! this is on my todo list for later. i will do it the way i did the other one.

https://github.com/Financial-Times/polyfill-library/actions/runs/2338900248

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,8192 @@
 {
   "name": "polyfill-library",
   "version": "4.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (romainmenke) [inline package-lock.json:4]: @plesiecki This changes the lock file version.
Can you apply this change while preserving the old one?

Node 12 (with corresponding npm version) is the one we still use here :)

Comment (chee) [reply package-lock.json:4]: eagle eyes @romainmenke

👁️ 👀 👁️ 
&nbsp; &nbsp; &nbsp; 🐽 
&nbsp; &nbsp; &nbsp; 👄

Comment (plesiecki) [reply package-lock.json]: thanks, done

**Comment by plesiecki** [comment]
I've fixed package.json. Are we going to run tests again? Previously tests passed.

**Comment by JakeChampion** [comment]
@plesiecki do you know what the breaking changes included in version 5 are?

**Comment by JakeChampion** [comment]
> @plesiecki do you know what the breaking changes included in version 5 are?

It looks like the breaking change is only to an eslnt rule

https://github.com/formatjs/formatjs/blob/main/packages/intl-datetimeformat/CHANGELOG.md#breaking-changes-1

**Comment by chee** [comment]
maybe we should add a test to https://github.com/Financial-Times/polyfill-library/blob/master/polyfills/Intl/DateTimeFormat/tests.js to ensure these return "Invalid Date", and make sure we don't break this later

**Comment by plesiecki** [comment]
Tests have been added ;)

**Comment by chee** [comment]
https://github.com/Financial-Times/polyfill-library/actions/runs/2382944743

**Comment by plesiecki** [comment]
@chee Let's try one more time 🙏

**Comment by plesiecki** [comment]
Phew, all test passed 🎉

**Review by chee — APPROVED** [review]
let's go

**Comment by plesiecki** [comment]
Looking forward to the release 🙏

**Comment by plesiecki** [comment]
@chee Please 🙏

**Comment by romainmenke** [comment]
@plesiecki Thank you again for this update and sorry for the delays :)

**Comment by plesiecki** [comment]
Lovely, looking forward to the release.

**Comment by romainmenke** [comment]
@plesiecki I fear we will have to revert this change because there seems to be a serious performance regression.

I've opened an issue upstream : https://github.com/formatjs/formatjs/issues/3747

_I might have jinxed it by saying it would get released next week_

**Comment by plesiecki** [comment]
Oh, that's a pity. Where can I see logs from failing tests?

**Comment by romainmenke** [comment]
Absolutely : https://github.com/Financial-Times/polyfill-library/runs/7694243715?check_suite_focus=true

Our test suite doesn't really any special handling or good error messages for these timeouts. But I noticed something was wrong the run in Internet Explorer suddenly took more than 20 minutes.

Screenshot 2022-08-06 at 22 48 58

Hopefully it gets resolved soon upstream so that we can merge the updates here :)

**Comment by longlho** [comment]
hmm it seems weird that IE9 ok but not IE10/11? I'd say perf regression is expected to a certain degree, since CLDR & IANA data only gets bigger over time.

**Pull request merged**